client/src/components/ReviewTile.js

```diff
@@ -12,11 +15,58 @@ const ReviewTile = (props) => {
     }
   })
 
+  const handleClick = async (event) => {
+    setShowVoteError(false)
```

Comment:
the default state is already false, do we need to reset state to its default here?